expressions/aggregation/AggregationHandleAvg.hpp

```diff
@@ -127,6 +170,9 @@ class AggregationHandleAvg : public AggregationConcreteHandle {
   void mergeStates(const AggregationState &source,
                    AggregationState *destination) const override;
 
+  void mergeStatesFast(const uint8_t *source,
```

Comment:
Can you please align the parameters? I use a vim plugin called clang-format which helps greatly in fixing all the style issues with minimal efforts. If you need help in configuring that plugin, please let me know. It's a big time saver.